src/components/flashMessage/flashMessage.module.css

```diff
@@ -3,7 +3,7 @@
   font-size: 1.1rem;
   color: var(--text-color);
   line-height: 1.25;
-  background-color: rgba(255, 255, 255, 0.85);
+  background-color: rgba(255, 255, 255, 0.95);
```

Comment:
This was just better, the `FlashMessage` text turned out to be hard to read when the background was noisy or colourful.